### PR TITLE
std.uri: Add unittests for 100% code coverage.

### PR DESCRIPTION
**Pull request by berni44**

### CODE REVIEW

File: std/uri.d [hunk 1]
```diff
@@ -172,6 +172,18 @@ private string URI_Encode(dstring str, uint unescapedSet) @safe pure
...
 
+@safe pure unittest
+{
+    import std.exception : assertThrown;
+
+    assert(URI_Encode("",0)=="");
+    assert(URI_Encode(URI_Decode("%F0%BF%BF%BF",0),0)=="%F0%BF%BF%BF");
+    dstring a;
+    a ~= cast(dchar) 0xFFFFFFFF;
+    assertThrown(URI_Encode(a,0));
```
Comment (atilaneves) [inline std/uri.d]: Spaces after commas and around the equals operator please.

Comment (berni44) [reply std/uri.d]: Strange, that the tests did not find this...

**Comment by burner** [comment]
squash and lets go.

**Comment by burner** [comment]
@wilzbach sweet didn't know that. thanks

**Pull request merged**